src/ISA/ISA/ArcTypes/ArcTables.fs

```diff
@@ -297,8 +304,52 @@ type ArcTables(thisTables:ResizeArray<ArcTable>) =
         |> List.map (fun (name,ps) ->
             ps
             |> List.collect (fun p -> ProcessParsing.processToRows p)
-            |> fun rows -> ProcessParsing.alignByHeaders rows
+            |> fun rows -> ProcessParsing.alignByHeaders true rows
             |> fun (headers, rows) -> ArcTable.create(name,headers,rows)
         )
         |> ResizeArray
+        |> ArcTables
+
+    static member updateReferenceTablesBySheets (referenceTables : ArcTables,sheetTables : ArcTables,?keepUnusedRefTables : bool) : ArcTables =
+        let keepUnusedRefTables = Option.defaultValue false keepUnusedRefTables
```

Comment:
Can we move this somewhere less visible?

Reply:
Will first make a static member out of it. We need a place for more complex operations that do not necessarily belong to a downstream repository.

Reply:
Whoops, this comment belongs to your comment above. Still my point remains.

Reply:
Maybe a module similiar to `module ARCtrl.ISA.IdentifierSetters` at the end of the arctypes folder with extensions?